### ISSUE TITLE
Remove linebreaks & indentation

### ISSUE THREAD

**Issue by NightkingCH** [ludohenin/gulp-inline-ng2-template#7]
Hi there

First of all, thanks for your work! Your plugin is exactly the one I needed!

Unfortunately you're adding linebreaks ***\n*** between this character ***`*** and unnecessary whitespaces , in my eyes ;), between the start and the end. Sadly this breaks my minified HTML. Finally typescripts screws it up by rewriting those breaks into "\n \uFFFF" which are interpreted in the end by the browser.

Is there a way add a flag or something which skips the beautification or to remove it? Temporary I rewrote it like this:

From
```
if ('html' === opts.type)  assetFiles = opts.prop + ': `\n' + assetFiles + '\n' + indentation + '`';
```

To
```
if ('html' === opts.type)  assetFiles = opts.prop + ': `' + assetFiles + '`';
```

And I also removed the logic for the spaces in the "indent"-function.

```
function indent(str) {
      var lines = [];
      str.split('\n').forEach(function (line) {
        lines.push(line);
      });
      return lines.join('\n');
    }
```

In my case these works. When I minify my HTML I get an one liner. When I skip my minification I get a human readable template string.

**Comment by ludohenin** [comment]
Do you minify your html files before inlining them ?

**Comment by ludohenin** [comment]
Fixed by #19
